pom.xml

```
@@ -74,8 +74,8 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.319.x</artifactId>
-                <version>1508.v4b_d09ff0e893</version>
+                <artifactId>bom-2.361.x</artifactId>
+                <version>1763.v092b_8980a_f5e</version>
```

Final release of bom-2.361.x
```suggestion
                <version>2102.v854b_fec19c92</version>
```